src/operator/convolution.cc

```diff
@@ -15,6 +20,19 @@ Operator* CreateOp<cpu>(ConvolutionParam param, int dtype,
                         std::vector<TShape> *out_shape,
                         Context ctx) {
   Operator *op = NULL;
+#if MXNET_USE_MKL2017 == 1
+  if ((param.dilate[0] == 1 && param.dilate[1] == 1)
+      && param.kernel.ndim() == 2) {
```

Comment:
Yes, MKL support grouping. we use groups conv API in mkl_convolution-inl.h like below:
dnnGroupsConvolutionCreateForward
dnnGroupsConvolutionCreateBackwardData